app/views/catalog/_index_split_default.html.erb

```diff
@@ -1,19 +1,27 @@
 <% # header bar for doc items in index view -%>
 <%= content_tag :div, class: 'documentHeader row', data: { layer_id: document.id, bbox: document.bounding_box_as_wsen } do %>
-  <div class='status-icons'>
-    <%= render partial: 'header_icons', locals: { document: document } %>
-  </div>
-  <h3 class="index_title col-sm-9s cosl-lg-10 text-span">
-    <span class='caret-toggle'></span>
-    <% counter = document_counter_with_offset(document_counter) %>
-    <span class="document-counter">
-      <%= t('blacklight.search.documents.counter', :counter => counter) if counter %>
+  <h3 class="index_title col pl-2">
```

Comment:
I wasn't sure if the non-cols that had the caret spilling over into the gutter was a feature or not, but I split the difference and continue to have them spill but not as much as to be inaccessible on mobile. 🤷‍♂️

Reply:
Thanks I think it is. This looks great.